src/cplscheme/MultiCouplingScheme.hpp

```diff
@@ -92,19 +93,44 @@ class MultiCouplingScheme : public BaseCouplingScheme {
 
   logging::Logger _log{"cplscheme::MultiCouplingScheme"};
 
+  /**
+   * @brief BiCouplingScheme has _sendData and _receiveData
+   * @returns DataMap with all data
+   */
+  DataMap getAllData() override
+  {
+    DataMap allData;
+    // @todo user C++17 std::map::merge
+    for (auto &sendData : _sendDataVector) {
+      for (auto &dataPair : sendData.second) {
+        if (!utils::contained(dataPair.first, allData)) {
+          allData.emplace(dataPair);
+        }
+      }
```

Comment:
You could simply use `allData.insert(sendData.second.begin(), sendData.second.end())`.

[Reference](https://en.cppreference.com/w/cpp/container/map/insert)

> Inserts element(s) into the container, if the container doesn't already contain an element with an equivalent key. 

Same below

Reply:
Done. 854ae5fabe13b0709de5b1282301fa818d3af0c6